rebar.config

```diff
@@ -23,7 +23,7 @@
 %% == Dependencies ==
 
 {deps, [
-  {chatterbox, "0.5.0"},
+  {chatterbox, {git, "https://github.com/joedevivo/chatterbox.git", {branch, "master"}}},
```

Comment:
It would be better to use something like `{ref, "f5b68963bc63b3b2cf8701fafe61cfae9c0a383f"}` here :)

Reply:
Yes, if we want to stay there. Usually I let the .lock file track these. But I can make this change if needed :)